yml/cdb_lsstcomcamsim.yaml

```diff
@@ -476,3 +475,776 @@ tables:
+  - name: s_region
+    "@id": "#visit1.s_region"
+    datatype: string
+    length: 1024
+    description: Sky region in STC-S format (https://www.ivoa.net/documents/STC-S/20130917/index.html).
+    ivoa:ucd: pos.outline;obs.field
+- name: visit1_quicklook
+  "@id": "#visit1_quicklook"
+  primaryKey:
+  - "#visit1_quicklook.visit_id"
+  constraints:
+  - name: fk_obs_id
+    "@id": "#visit1_quicklook.fk_obs_id"
+    "@type": ForeignKey
+    description: Quicklook visit_id must be an Exposure exposure_id.
+    columns: ["#visit1_quicklook.visit_id"]
+    referencedColumns: ["#exposure.exposure_id"]
+  columns:
+  - name: visit_id
+    "@id": "#visit1_quicklook.visit_id"
+    datatype: long
+    nullable: false
+    description: Unique identifier.
+  - name: astrom_offset_mean_min
+    "@id": "#visit1_quicklook.astrom_offset_mean_min"
+    datatype: float
+    description: Mean offset of astrometric calibration matches (minimum across all detectors).
+    ivoa:unit: arcsec
+  - name: astrom_offset_mean_max
+    "@id": "#visit1_quicklook.astrom_offset_mean_max"
+    datatype: float
+    description: Mean offset of astrometric calibration matches (maximum across all detectors).
+    ivoa:unit: arcsec
+  - name: astrom_offset_mean_median
+    "@id": "#visit1_quicklook.astrom_offset_mean_median"
+    datatype: float
+    description: Mean offset of astrometric calibration matches (median across all detectors).
+    ivoa:unit: arcsec
+  - name: astrom_offset_std_min
+    "@id": "#visit1_quicklook.astrom_offset_std_min"
+    datatype: float
+    description: Standard deviation of offsets of astrometric calibration matches (minimum across all detectors).
+    ivoa:unit: arcsec
+  - name: astrom_offset_std_max
+    "@id": "#visit1_quicklook.astrom_offset_std_max"
+    datatype: float
+    description: Standard deviation of offsets of astrometric calibration matches (maximum across all detectors).
+    ivoa:unit: arcsec
+  - name: astrom_offset_std_median
+    "@id": "#visit1_quicklook.astrom_offset_std_median"
+    datatype: float
+    description: Standard deviation of offsets of astrometric calibration matches (median across all detectors).
+    ivoa:unit: arcsec
+  - name: eff_time_min
+    "@id": "#visit1_quicklook.eff_time_min"
+    datatype: float
+    description: Effective exposure time (minimum across all detectors).
+    ivoa:unit: s
+  - name: eff_time_max
+    "@id": "#visit1_quicklook.eff_time_max"
+    datatype: float
+    description: Effective exposure time (maximum across all detectors).
+    ivoa:unit: s
+  - name: eff_time_median
+    "@id": "#visit1_quicklook.eff_time_median"
+    datatype: float
+    description: Effective exposure time (median across all detectors).
+    ivoa:unit: s
+  - name: eff_time_psf_sigma_scale_min
+    "@id": "#visit1_quicklook.eff_time_psf_sigma_scale_min"
+    datatype: float
+    description: Eff time PSF sigma scale (minimum across all detectors).
```

Comment:
description: Eff time PSF sigma scale (minimum across all detectors).

All instances of "eff time" in the description can just be replaced with "Effective exposure time". It's weird, but it's a thing, and I don't think there's better words to use without it getting impossibly long. People who care about it will already know what it is, and people who don't understand can either ask or leave it alone.